test/FunctionalTests/Framework/Workspaces/APICallLog.cs

```diff
@@ -546,23 +539,6 @@ public void ChangeInterceptor<T>(DataService<T> service, string methodName, obje
     }
     #endregion
 
-    #region IExpandProvider
-#pragma warning disable 618 // Disable "obsolete" warning for the IExpandProvider interface. Used for backwards compatibilty.
-    public class IExpandProviderCallLog : InterfaceCallLog<IExpandProvider>

```

Comment:
How come this didn't have the `Obsolete` attribute. Are you sure it's obsolete?

Reply:
The `IExpandProvider` that it implements is the one that's obsolete.